spec/requests/registrations_spec.rb

```diff
@@ -137,10 +137,11 @@
         allow(FeatureFlag).to receive(:enabled?).with(:creator_onboarding).and_return(true)
         allow(FeatureFlag).to receive(:enabled?).with(:runtime_banner).and_return(false)
         allow(SiteConfig).to receive(:waiting_on_first_user).and_return(true)
+        allow(Settings::UserExperience).to receive(:public).and_return(false)
       end
 
       it "renders the creator onboarding form" do
-        get new_user_registration_path
+        get root_path
```

Comment:
This is the reason the problem went undetected. Creator onboardings happen on root_path, within `ApplicationController` when it renders the form instead of forwarding it to `RegistrationsController`

Reply:
Nice find! 💯